client/blocks/category-pill-navigation/README.md

```diff
@@ -19,11 +19,13 @@ function render() {
 			selectedCategory="category-2"
 			buttons={ [
 				{
+					id: 'discover',
```

Comment:
Add please the info about `id` to the bottom of this file, with explanations of the properties

Reply:
Added!